cli/command/network/connect.go

```diff
@@ -38,11 +38,11 @@ func newConnectCommand(dockerCli *command.DockerCli) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVar(&opts.ipaddress, "ip", "", "IP Address")
-	flags.StringVar(&opts.ipv6address, "ip6", "", "IPv6 Address")
+	flags.StringVar(&opts.ipaddress, "ip", "", "Container IPv4 address (e.g. 172.30.100.104)")
+	flags.StringVar(&opts.ipv6address, "ip6", "", "Container IPv6 address (e.g. 2001:db8::33)")
```

Comment:
Wondering if we should change it the other way round; `Container ...` seems a bit redundant on `docker run`, so we could remove it there. For `network connect` we could use something like `IPv6 address for the container (e.g., ....)` ("for the container" could be useful information on `network connect`)

@aboch wdyt?